packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -45,7 +44,6 @@ export class ZipkinExporter implements SpanExporter {
     const urlStr = config.url || ZipkinExporter.DEFAULT_URL;
     const urlOpts = url.parse(urlStr);
 
-    this._forceFlush = config.forceFlush || true;
```

Comment:
I think we should remove `_forceFlushOnShutdown` from Jaeger exporter also.

Reply:
In zipkin this functionality was in fact "dead" but for Jaeger it does something - there is a timeout which waits for the Jaeger to close the sender. Are you ok to make it in separate PR then  ?

Reply:
sure 👍